psi4/src/psi4/libscf_solver/hf.h

```diff
@@ -213,6 +213,8 @@ class HF : public Wavefunction {
 
     /** Computes the initial MO coefficients (default is to call form_C) */
     virtual void form_initial_C() { form_C(); }
+    /** Computes the initial Fock matrix (default is to call form_F) */
+    virtual void form_initialF() { form_F(); }
```

Comment:
Can this be `form_initial_F` to conform to other similar signatures?

Reply:
Sure. I just took the name that already existed in the sources, after finding out that I was adding what was basically a duplicate copy... I'll rename it.